src/main/java/com/dreammaster/bartworksHandler/BioItemLoader.java

```diff
@@ -50,100 +55,79 @@ public class BioItemLoader {
+
         BIOTEMSSTACKS[0].stackSize = 1;
-        RA.addChemicalRecipe(
-                Materials.MeatRaw.getDust(1),
-                Materials.Bone.getDust(2),
-                Materials.DilutedSulfuricAcid.getFluid(500),
-                Materials.Water.getFluid(500),
-                BIOTEMSSTACKS[0],
-                800,
-                480);
+
+        GT_Values.RA.stdBuilder().itemInputs(Materials.MeatRaw.getDust(1), Materials.Bone.getDust(2))
+                .itemOutputs(BIOTEMSSTACKS[0]).fluidInputs(Materials.DilutedSulfuricAcid.getFluid(500))
+                .fluidOutputs(Materials.Water.getFluid(500)).duration(40 * SECONDS).eut(TierEU.RECIPE_HV)
+                .addTo(UniversalChemical);
+
         BIOTEMSSTACKS[0].stackSize = 4;
-        RA.addChemicalRecipe(
-                BIOTEMSSTACKS[0],
-                Materials.Water.getCells(3),
-                Materials.PhosphoricAcid.getFluid(1000),
-                new FluidStack(BIOFLUIDS[0], 4000),
-                Materials.Empty.getCells(3),
-                1600,
-                480);
-        RA.addChemicalRecipe(
-                BIOTEMSSTACKS[0],
-                Materials.PhosphoricAcid.getCells(1),
-                Materials.Water.getFluid(3000),
-                new FluidStack(BIOFLUIDS[0], 4000),
-                Materials.Empty.getCells(1),
-                1600,
-                480);
+
+        GT_Values.RA.stdBuilder().itemInputs(BIOTEMSSTACKS[0], Materials.Water.getCells(3))
+                .itemOutputs(Materials.Empty.getCells(3)).fluidInputs(Materials.PhosphoricAcid.getFluid(1000))
+                .fluidOutputs(new FluidStack(BIOFLUIDS[0], 4000)).duration(1 * MINUTES + 20 * SECONDS)
+                .eut(TierEU.RECIPE_HV).addTo(UniversalChemical);
+
+        GT_Values.RA.stdBuilder().itemInputs(BIOTEMSSTACKS[0], Materials.PhosphoricAcid.getCells(1))
+                .itemOutputs(Materials.Empty.getCells(1)).fluidInputs(Materials.Water.getFluid(3000))
+                .fluidOutputs(new FluidStack(BIOFLUIDS[0], 4000)).duration(1 * MINUTES + 20 * SECONDS)
+                .eut(TierEU.RECIPE_HV).addTo(UniversalChemical);
+
         BIOTEMSSTACKS[0].stackSize = 1;
         BIOTEMSSTACKS[1].stackSize = 4;
-        RA.addCentrifugeRecipe(
-                GT_Utility.getIntegratedCircuit(1),
-                NI,
-                new FluidStack(BIOFLUIDS[0], 6000),
-                NF,
-                Materials.Phosphorus.getDust(1),
-                BIOTEMSSTACKS[1],
-                NI,
-                NI,
-                NI,
-                NI,
-                null,
-                2400,
-                480);
+
+        GT_Values.RA.stdBuilder().itemInputs(GT_Utility.getIntegratedCircuit(1))
+                .itemOutputs(Materials.Phosphorus.getDust(1), BIOTEMSSTACKS[1])
+                .fluidInputs(new FluidStack(BIOFLUIDS[0], 6000)).noFluidOutputs().duration(2 * MINUTES)
+                .eut(TierEU.RECIPE_HV).addTo(sCentrifugeRecipes);
```

Comment:
It seems like the itemOutputs are not copied,  so we need to change the stacksize in a difftent way.

![image](https://github.com/GTNewHorizons/NewHorizonsCoreMod/assets/53441451/e3fe46c5-62d2-4758-aebe-f9853035483d)